dist/images/ovnkube.sh

```diff
@@ -775,12 +775,20 @@ ovn-node () {
   wait_for_event process_ready ovn-controller
   sleep 1
 
+  OVN_ENCAP_IP=""
+  ovn_encap_ip=`ovs-vsctl --if-exists get Open_vSwitch . external_ids:ovn-encap-ip | tr -d '\"'`
```

Comment:
You can pass `--data=bare` to ovs-vsctl instead of piping to `tr`

Reply:
Right, I tried that. However, if an user has provided the ovn-encap-ip with quotes, then --data=bare will not work. For example:

```
$ ovs-vsctl list open |grep external_ids
external_ids        : {hostname="sdn-test1", ovn-bridge-mappings="physnet:breno1", ovn-encap-ip="10.8.51.51", ovn-encap-type=geneve, ovn-nb="tcp:10.8.51.51:6641", ovn-openflow-probe-interval="180", ovn-remote="tcp:10.8.51.51:6642", ovn-remote-probe-interval="100000", rundir="/var/run/openvswitch", system-id="3b3e3241-2435-425d-bccf-d794465103dc"}

$ ovs-vsctl --data=bare get Open_vSwitch . external_ids:ovn-encap-ip
"10.8.51.51"
```